mne/decoding/time_gen.py

```diff
@@ -1076,6 +1090,8 @@ class TimeDecoding(_GeneralizationAcrossTime):
                 one time sample.
 
         If None, empty dict.
+    predict_function : str
+        Function used to make estimator predictions. Defaults to 'predict'.
```

Comment:
I think the docs should make explicit what's actually going on. So something like: "The method used to make predictions by calling a method of this name on each estimator on each time point per trial. Must be a method of the estimator and follow the scikit-learn predict API. Must return values compatible with the input of the scorer. Examples: for `sklearn.linear_model.LogisticRegression`, `predict` is supported, and `predict_proba` is supported if the scorer can deal with two values per prediction. Defaults to 'predict'."